sysutils/rawhide/Portfile

```
@@ -20,16 +20,21 @@ license             GPL-3+
 categories          sysutils
 depends_lib         port:pcre2 port:libmagic
 maintainers         {raf.org:raf @macportsraf} openmaintainer
-revision            0
+revision            1
 
 github.setup        raforg ${name} ${version} v
```

This line should read:
```suggestion
github.setup        raforg rawhide 3.2 v
```
and then the separate `name` and `version` lines above should be removed.

Right after the `github.setup` line (or the `version` line, for ports that use that) is a good place to put the `revision` line.

---

The `github.setup` line typically goes near the top of the portfile, where you currently have the `name` and `version` lines.

---

I'm working on it, however what to do about the broken `fstatat` now (or just fix the legacysupport linking this time)?

---

@ryandesign: @macportsraf explained his preference for why he kept separate `name` and `version` lines aside from `github.setup` here: https://github.com/macports/macports-ports/pull/19042#discussion_r1225035115

---

Regarding fstatat() and the other POSIX 2008 functions: The code works on Linux, FreeBSD, OpenBSD, NetBSD, Solaris, later macOS (that doesn't need legacysupport), and Cygwin, so I don't think I'm using the functions incorrectly. fstatat() reports a bad file descriptor, but it's a positive file descriptor that was returned by openat(). I don't know what could be wrong with it. It's set to close on exec, but it's unlikely that that matters.

---

@catap Maybe you have an idea here? The issue is not ppc-specific, so Intel can be tested.